3.11/appendix-deprecated.md

```diff
@@ -32,6 +32,12 @@ considered obsolete and may get removed in a future release. They are currently
 kept for backward compatibility. There are usually better alternatives to
 replace the old features with:
 
+- **Custom Pregel**:
+  The experimental feature "Custom Pregel" has been removed.
```

Comment:
Not sure if this removal notice belongs to the deprecated features list. For example, I didn't see a removal notice for the async feature being added to this file - ref: https://github.com/arangodb/docs/pull/1221/files

@Simran-B please clarify.

Reply:
It's not intended as a changelog, but rather as a page we can redirect removed URLs to. There are no hard rules but the convention is to add larger features here when they are deprecated/removed.